presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaEndpointIdentificationAlgorithm.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka;
+
+public enum KafkaEndpointIdentificationAlgorithm
+{
+    HTTPS("https"),
+    DISABLED("");
+
+    public final String name;
+
+    @Override
+    public String toString()
+    {
+        return name;
+    }
+
+    KafkaEndpointIdentificationAlgorithm(String name)
+    {
+        this.name = name;
+    }
```

Comment:
Please do add:
```
    public static KafkaEndpointIdentificationAlgorithm fromString(String value)
    {
        return KafkaEndpointIdentificationAlgorithm.valueOf(value.toUpperCase(ENGLISH));
    }
```
So one can specify value in config property lowercased.

Reply:
won't suffice for `""` to disable it, but will add a check for that as well

Reply:
I think that in presto we should require explicit setting "disabled" or "https". Or just skip config property for default.

Reply:
`""` is used in `kafka-clients` but can change this here to use `disabled`...

Reply:
As for the value we send to kafka-clients I have no strong opinion. Whichever works. I was just saying about expressing intent in Presto configuration.